### PR TITLE
Tile/Entity name 0.16 port

### PR DESCRIPTION
**Pull request by Valansch**

### CODE REVIEW

File: locale/gen_combined/grilledham_map_gen/builders.lua [hunk 1]
```diff
@@ -95,9 +95,9 @@ local tile_map =
     [24] = "red-desert-1",
     [25] = "red-desert-2",
     [26] = "red-desert-3",
-    [27] = "sand-1",
-    [28] = "sand-2",
-    [29] = "sand-3",
+    [27] = "sand-1-1",
+    [28] = "sand-1-2",
+    [29] = "sand-1-3",
     [30] = "stone-path",
```
Comment (grilledham) [inline locale/gen_combined/grilledham_map_gen/builders.lua]: These changes seem like a mistake.

File: locale/gen_combined/red_planet.lua [hunk 2]
```diff
@@ -287,8 +287,8 @@ function run_combined_module(event)
 			local noise_terrain_5 = perlin:noise(((pos_x+seed)/5),((pos_y+seed)/5),0)					
 			noise_terrain_5 = noise_terrain_5 * 4
 			seed = seed + seed_increment_number
-			local noise_sand = perlin:noise(((pos_x+seed)/18),((pos_y+seed)/18),0)					
-			noise_sand = noise_sand * 10
+			local noise_sand-1 = perlin:noise(((pos_x+seed)/18),((pos_y+seed)/18),0)					
+			noise_sand-1 = noise_sand-1 * 10
```
Comment (grilledham) [inline locale/gen_combined/red_planet.lua]: The name of the variable doesn't need changing. But this won't break anything so it's fine.

File: locale/gen_combined/grilledham_map_gen/builders.lua (hunk 1 above)
Comment (Valansch) [reply locale/gen_combined/grilledham_map_gen/builders.lua]: yes, will revert that. Was just running a search and replace over the whole locale folder.

File: locale/gen_combined/red_planet.lua (hunk 2 above)
Comment (grilledham) [reply locale/gen_combined/red_planet.lua]: if you are going to change "noise_sand-1" back to "noise_sand" make sure you have change all instances in red_planet.lua and red_planet_v2.lua

Comment (Valansch) [reply locale/gen_combined/red_planet.lua]: done. I hope i got them all.

**Comment by Valansch** [comment]
Open for review :)

**Pull request merged**